docs/editors/mypy.md

```diff
@@ -0,0 +1,48 @@
+# MYPY Plugin Setup
+
+Mypy is a static type checker for Python 3 and Python 2.7. If you sprinkle your code with type annotations, mypy can type check your code and find common bugs.
+
+## Installing and running mypy
+
+Mypy requires Python 3.6 or later to run
+
+`$ python3 -m pip install mypy`
+
+Once mypy is installed, run it by using the mypy tool:
+
+- `$ mypy program.py` (Python 3)
+
+- `$ mypy --py2 program.py` (Python 2)
+
+This command makes mypy type check your program.py file and print out any errors it finds. Mypy will type check your code statically: this means that it will check for errors without ever running your code, like a linter.
+
+A function without type annotations is considered to be dynamically typed by mypy:
+
+```python
+def greeting(name):
+    return 'Hello ' + name
+```
+
+By default, mypy will not type check dynamically typed functions. This means that with a few exceptions, mypy will not report any errors with regular unannotated Python.
+
+You can teach mypy to detect these kinds of bugs by adding type annotations (also known as type hints).
+
+```python
+def greeting(name: str) -> str:
+    return 'Hello ' + name
+```
+
+This function is now statically typed: mypy can use the provided type hints to detect incorrect usages of the `greeting` function. For example, it will decline the following calls since the arguments have invalid types:
+
+```python
+def greeting(name: str) -> str:
+    return 'Hello ' + name
+
+greeting(3)         # Argument 1 to "greeting" has incompatible type "int"; expected "str"
+greeting(b'Alice')  # Argument 1 to "greeting" has incompatible type "bytes"; expected "str"
+```
+
+If you are trying to type check Python 2 code, you can add type hints using a comment-based syntax instead of the Python 3 annotation syntax.
+
+**Note -**
+You are always free to ignore the errors mypy reports and treat them as warnings, if you so wish: mypy runs independently from Python itself.
```

Comment:
I think instead of writing how to get started with mypy we can link the users to their docs: https://mypy.readthedocs.io/en/stable/getting_started.html

And we can mention how to enable the mypy plugin for strawberry, like we did in the the Readme:

https://github.com/strawberry-graphql/strawberry#type-checking

Reply:
@patrick91 Now It's more clear. I created a mypy tutorial file, instead, I should've done a mypy Plugin setup. Thanks for the review btw.